consul/outputs.tf

```diff
@@ -1,9 +1,9 @@
 output "address" {
-  value = "public.consul.${var.environment}.${var.region}.${var.account}.${var.nubis_domain}:443"
+  value = "localhost:8500"
 }
 
 output "scheme" {
-  value = "https"
+  value = "http"
```

Comment:
Why is this changing to http?

Reply:
This is because the CI system is talking to local consul, does not go across the network at all and the local consul does not have a local https endpoint. This is a change as we are no longer doing cross VPC connections.